libs/openFrameworks/gl/ofShader.cpp

```diff
@@ -381,29 +427,13 @@ void ofShader::checkProgramInfoLog(GLuint program) {
 	if (infoLength > 1) {
 		GLchar* infoBuffer = new GLchar[infoLength];
 		glGetProgramInfoLog(program, infoLength, &infoLength, infoBuffer);
+		// TODO: it appears that Raspberry Pi only reports shader errors whilst linking,
+		// but then it becomes hard to figure out whether the fragment or the
+		// vertex shader caused the error.
+		// We need to find a robust way of extracing this information from
+		// the log, and unfortunately can't use regex whilst gcc on RPi is assumed to
+		// be < 4.9, which is the first version fully supporting this c++11 feature.
 		string msg = "ofShader: program reports:\n";
-#ifdef TARGET_RASPBERRYPI
```

Comment:
Yes, it would be great to have something in, but i'm not too sure about how we'd catch errors from within the vertex shader - the removed code assumed hardcoded that any error must have come from the fragment shader source. 
I figure it's better to print the straight error log rather than maybe confusing listings from the wrong file.